resolwe_bio/processes/import_data/image.yml

```diff
@@ -0,0 +1,43 @@
+# =================
+# Import image file
+# =================
+---
+
+- slug: upload-image-file
+  name: Image file
+  data_name: '{{ src.file|default:"?" }}'
+  version: 0.0.1
+  type: data:file:image
+  category: upload
+  persistence: RAW
+  description: >
+    Upload image file.
+  input:
+    - name: src
+      label: Image file
+      type: basic:file
+      description: >
+        Image file.
+      validate_regex: '\.(tiff|tif|jpg|jpeg|png|TIFF|TIF|JPG|JPEG|PNG)$'
+  output:
+    - name: file
+      label: Image file
+      type: basic:file
+  run:
+    runtime: polyglot
+    bash: |
+      if [[ '.{{ src.file }}' =~ \.(gif|GIF)$ ]]; then
```

Comment:
I would not use re-import here. Just move the file in place as is. re-import only makes sense when you wish to unify the extensions or handle compression. None of this is necessary here.